### PR TITLE
add tracing to Data proxy limitations

### PR DESCRIPTION
**Pull request by Jolg42**
Since `tracing` is not compatible with the Data Proxy, I added it in the limitations

### CODE REVIEW

**Review by zachtil — APPROVED** [review]
Thanks for starting this, @Jolg42! And thanks for the explanations around the `previewFeatures` line in `prisma.schema`.

I committed a bunch of changes to stack the preview features in a bullet and demo what the `prisma.schema` changes need to be.

File: content/800-data-platform/03-data-proxy.mdx [hunk 1]
```diff
@@ -153,7 +153,24 @@ You are now ready to try the Data Proxy. Go ahead and start your app!
...
+
+```diff
+   generator client {
+     provider        = "prisma-client-js"
+-     previewFeatures = ["interactiveTransactions", "metrics", "tracing"]
++     previewFeatures = []
+   }
+```
+
+When you run `prisma generate --data-proxy`, you will receive an error if you have one of the listed preview features enabled.
```
Comment (andrew-walford-prisma) [inline content/800-data-platform/03-data-proxy.mdx]:
```suggestion
When you run `prisma generate --data-proxy`, you will receive an error if you have one or more of the listed preview features enabled.
```

File: content/800-data-platform/03-data-proxy.mdx [hunk 2]
```diff
@@ -153,7 +153,24 @@ You are now ready to try the Data Proxy. Go ahead and start your app!
...
+
+```diff
+   generator client {
+     provider        = "prisma-client-js"
+-     previewFeatures = ["interactiveTransactions", "metrics", "tracing"]
++     previewFeatures = []
+   }
+```
+
+When you run `prisma generate --data-proxy`, you will receive an error if you have one or more of the listed preview features enabled.
```
Comment (andrew-walford-prisma) [inline content/800-data-platform/03-data-proxy.mdx]:
```suggestion
When you run `prisma generate --data-proxy`, you will receive an error if you have one or more of these preview features enabled.
```

**Review by andrew-walford-prisma — COMMENTED** [review]
LGTM with one suggestion, which I'll commit

**Pull request merged**